apps/ui/src/views/Auction/Bids.vue

```diff
@@ -1 +1,180 @@
-<template>Bids</template>
+<script setup lang="ts">
+import { useInfiniteQuery, useQuery } from '@tanstack/vue-query';
+import { AuctionNetworkId, formatPrice, getOrders } from '@/helpers/auction';
+import {
+  AuctionDetailFragment,
+  OrderDetailFragment
+} from '@/helpers/auction/gql/graphql';
+import { getTokenPrices } from '@/helpers/coingecko';
+import { CHAIN_IDS, COINGECKO_ASSET_PLATFORMS } from '@/helpers/constants';
+import { _c, _n, _p, _t, shortenAddress } from '@/helpers/utils';
+
+const LIMIT = 20;
+
+const props = defineProps<{
+  network: AuctionNetworkId;
+  auctionId: string;
+  auction: AuctionDetailFragment;
+}>();
+
+const {
+  data,
+  fetchNextPage,
+  hasNextPage,
+  isPending,
+  isFetchingNextPage,
+  isError
+} = useInfiniteQuery({
+  initialPageParam: 0,
+  queryKey: ['auction', () => props.network, () => props.auctionId, 'orders'],
+  queryFn: ({ pageParam }) =>
+    getOrders(props.auctionId, props.network, {
+      first: LIMIT,
+      skip: pageParam
+    }),
+  getNextPageParam: (lastPage, pages) => {
+    if (lastPage.length < LIMIT) return null;
+
+    return pages.length * LIMIT;
+  }
+});
+
+const { data: biddingTokenPrice, isLoading: isBiddingTokenPriceLoading } =
+  useQuery({
+    queryKey: [
+      'auction',
+      () => props.network,
+      () => props.auctionId,
+      'pricing'
+    ],
+    queryFn: async () => {
+      const chainId = CHAIN_IDS[props.network];
+      if (!(chainId in COINGECKO_ASSET_PLATFORMS)) {
+        return 0;
+      }
+
+      const coingeckoAssetPlatform =
+        COINGECKO_ASSET_PLATFORMS[
+          chainId as keyof typeof COINGECKO_ASSET_PLATFORMS
+        ];
+
+      const coins = await getTokenPrices(coingeckoAssetPlatform, [
+        props.auction.addressBiddingToken
+      ]);
+
+      return (
+        coins[props.auction.addressBiddingToken.toLocaleLowerCase()]?.usd ?? 0
+      );
+    }
+  });
+
+async function handleEndReached() {
+  if (!hasNextPage.value) return;
+
+  fetchNextPage();
+}
+
+function getOrderPercantage(order: OrderDetailFragment) {
+  return Number(order.buyAmount) / Number(props.auction.exactOrder.sellAmount);
+}
+</script>
+
+<template>
+  <div>
+    <UiSectionHeader label="Bids" sticky />
```

Comment:
First column should align to header

<img width="1170" height="794" alt="Image" src="https://github.com/user-attachments/assets/479108f6-0cb7-4cb8-9f43-b50bda1f1564" />

Leaderboard
<img width="1070" height="830" alt="Image" src="https://github.com/user-attachments/assets/2e28281d-db2a-49e0-a13c-036424aeef3d" />